content/ar/courses/350/blog-project/models.md

```diff
@@ -1,117 +1,118 @@
 ---
-# Page title
-title: The Blog Project Models
+#عنوان الصفحة
+العنوان: نماذج مشروع المدونة
 
-# Title for the menu link if you wish to use a shorter link title, otherwise remove this option.
-linktitle: Models
 
-# Date page published
-date: 2021-03-23
+# عنوان رابط القائمة إذا كنت ترغب في استخدام عنوان ارتباط أقصر ، وإلا فقم بإزالة هذا الخيار .
+عنوان الارتباط: النماذج
 
-# Academic page type (do not modify).
-type: book
+# تاريخ نشر الصفحة
+التاريخ: 2021-03-23
 
-# Position of this page in the menu. Remove this option to sort alphabetically.
-weight: 2
+# نوع الصفحة الأكاديمية (لا تعدل).
+النوع: كتاب
 
-draft: False
+#مكان هذه الصفحة في القائمة. قم بإزالة هذا الخيار للترتيب أبجديًا.
+الوزن: 2
+
+مسودة: خطأ
 
 
 ---
 
-{{% callout note %}}
-Switch to branch **2-models** from **malmarz/isom350-blog** github repo to see this step's implementation
-{{% /callout %}}
+{{% مذكرة شرح %}}
```

Comment:
@Naeimah do not translate anything inside {{ }} marks .. please return them back to english they are code